bootstrap/bootstrap.sh

```diff
@@ -0,0 +1,162 @@
+  echo "                              The image name must match the regex [${ALLOWED_IMAGE_NAME_CHARS}]+ (e.g. 'my-kurtosis-lambda-image')."
+  echo ""
+  exit 1 # Exit with an error so CI fails if this was accidentally called
+}
+
+# =============================================================================
+#                           Arg Parsing & Validation
+# =============================================================================
+lang="${1:-}"
+output_dirpath="${2:-}"
+lambda_image="${3:-}"
+
+if [ -z "${lang}" ]; then
+  echo "Error: Lang cannot be empty" >&2
+  show_help_and_exit
+fi
+if ! grep -q "^${lang}$" "${supported_langs_filepath}"; then
+  echo "Error: Unrecognized lang '${lang}'" >&2
+  show_help_and_exit
+fi
+if [ -z "${output_dirpath}" ]; then
+  echo "Error: Output dirpath must not be empty" >&2
+  show_help_and_exit
+fi
+if [ -d "${output_dirpath}" ] && [ "$(ls -A "${output_dirpath}")" ]; then
+  echo "Error: Output directory '${output_dirpath}' exists, but is not empty"
+  show_help_and_exit
+fi
+if [ -z "${lambda_image}" ]; then
+  echo "Error: Kurtosis Lambda image cannot be empty" >&2
+  show_help_and_exit
+fi
+sanitized_image="$(echo "${lambda_image}" | sed "s|[^${ALLOWED_IMAGE_NAME_CHARS}]||g")"
+if [ "${sanitized_image}" != "${lambda_image}" ]; then
+  echo "Error: Kurtosis Lambda image name '${lambda_image}' doesn't match regex [${ALLOWED_IMAGE_NAME_CHARS}]+" >&2
+  show_help_and_exit
+fi
+
+# =============================================================================
+#                                 Main Code
+# =============================================================================
+# Use language-specific prep script to populate contents of output directory
+if ! mkdir -p "${output_dirpath}"; then
+  echo "Error: Could not create output directory '${output_dirpath}'" >&2
+  exit 1
+fi
+lang_dirpath="${repo_root_dirpath}/${lang}"
+lang_bootstrap_dirpath="${script_dirpath}/${lang}"
+prep_new_repo_script_filepath="${lang_bootstrap_dirpath}/${PREP_NEW_REPO_FILENAME}"
+if ! bash "${prep_new_repo_script_filepath}" "${lang_dirpath}" "${output_dirpath}"; then
+  echo "Error: Failed to prep new repo using script '${prep_new_repo_script_filepath}'" >&2
+  exit 1
+fi
+
+# README file
+output_readme_filepath="${output_dirpath}/${OUTPUT_README_FILENAME}"
+cat <<EOF >"${output_readme_filepath}"
+My Kurtosis Lambda
+=====================
+Welcome to your new Kurtosis Lambda! You can use Example Kurtosis Lambda implementation as a pattern to create your own Kurtosis Lambda.
+
+1. Customize your own Kurtosis Lambda editing the generated files inside the `/path/to/your/code/repos/kurtosis-lambda/impl` folder
+    1. Rename files and objects, if you want, using a name that describes the functionality of your Kurtosis Lambda
+    1. Write the functionality of your Kurtosis Lambda inside the `execute` method
```

Comment:
Note on all this: we definitely don't want ot duplicate instructions, and right now the instructions for customizing the testsuite are duplicated both here and in the README. I'd prefer if, in the README of this repo (up above), we do like Kurtosis Libs does and says "follow the bootstrap instructions" rather than duplicating the steps for customizing.

That way, the flow can be:
1. Users read the README of Kurtosis Lambda Starter Pack
2. Users run the bootstrap
3. Bootstrap generates a README file for htem
4. Bootstrap directs them to read the README

and we only have the customization steps in one place.